docs/UserGuide.md

```diff
@@ -150,556 +157,1347 @@ An example of the final command is displayed below.
+  * Tags: `Supplies Flour`
+
+</div>
+
+### 2.2.2 Editing an order: `edit_order`
+
+Edit an existing order’s information.
+
+Syntax: `edit_order INDEX [on/ORDER_ITEM] [q/QUANTITY] [d/DEADLINE] [n/CUSTOMER_NAME] [a/CUSTOMER_ADDRESS] [p/CUSTOMER_PHONE_NUMBER] [s/STATUS]`
+
+<div markdown="block" class="alert alert-tip">
+
+:information_source: **Information**
+
+* Statuses available for setting are
+  * `N` or `n` for `Not Delivered`
+  * `I` or `i` for `In Progress`
+  * `D` or `d` for `Delivered`
+
+</div>
+
+<div markdown="block" class="alert alert-example">
+
+:clipboard: **Example 1: Standard command**
+
+* `edit_order 1 p/91234567 d/05/05/2024`<br>
+  Edits the **first** order information with the following details:
+  * Customer Phone Number: `91234567`
+  * Deadline: `05/05/2024`
+
+:clipboard: **Example 2: Shortcut command updating status**
+
+* `edit_o 3 q/20 s/I`<br>
+  Edits the **third** order information with the following details:
+  * Quantity: `3`
+  * Status: `In Progress`
+
+</div>
+
+### 2.2.3 Editing a task: `edit_task`
+
+Edit an existing task’s information.
+
+Syntax: `edit_task INDEX [n/TASK_NAME] [d/DEADLINE] [s/STATUS]`
+
+<div markdown="block" class="alert alert-tip">
+
+:information_source: **Information**
+
+* Statuses available for setting are
+  * `N` or `n` for `Not Done`
+  * `D` or `d` for `Done`
 
-Examples:
+</div>
+
+<div markdown="block" class="alert alert-example">
+
+:clipboard: **Example 1: Standard command**
 
-* `edit_supplier 1 n/Johnny p/90138482 t/` edits the 1st supplier's name to `Johnny`, phone number to `90138482` and removed all of its tags
-* `edit_s 3 t/Supplies Flour e/mark@example.com` replaced the 3rd supplier's tags to `Supplies Flour` and edited its email to `mark@example.com`
+* `edit_task 1 n/Get creamer`<br>
+  Edits the **first** task information with the following details:
+  * Task Name: `Get creamer`
 
-### Editing an order: `edit_order`
+:clipboard: **Example 2: Shortcut command updating status**
 
-Edits an order that is present in the order list.
+* `edit_t 3 d/31/12/2024 s/N`<br>
+  Edits the **third** task information with the following details:
+  * Deadline: `31/12/2024`
+  * Status: `Not Done`
 
-Syntax: `edit_order INDEX [n/CUSTOMER_NAME] [l/CUSTOMER_LOCATION] [p/CUSTOMER_PHONE_NUMBER] [e/CUSTOMER_EMAIL] [d/DEADLINE] [q/QUANTITY] [on/ORDER_NAME] [s/STATUS]…​`
+</div>
+
+### 2.2.4 Editing a menu item: `edit_item`
 
-* Edits the order at the specific `INDEX`. The index refers to the number shown in the orders list displayed. The index must be a positive integer 1, 2, 3, …
-* You have to key in at least one of the optional fields.
-* Existing values will be replaced with the input values.
-* When editing remarks, the existing remarks of the order will be removed and replaced with the given remarks (editing of tags is not accumulative).
-* You can remove the remarks by typing r/ without specifying any remarks after it.
+Edit an existing menu item’s information.
```

Comment:
Can specify that it doesn’t update total profits and total revenue because it does not make financial sense.